biztalk/core/operational-data-service.md

```diff
@@ -48,7 +48,7 @@ The feed includes the following data tables:
 ## Use Power BI to Load Operational Data.
 
 1. Download and install the [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on your BizTalk Server. You can select to open it, which is optional. If you have a work or school account, you may have access to Power BI. Try signing in with that account. Or, you can try it for free after signing up.
-2. Open the `\Program Files (x86)\Microsoft BizTalk Server 2016\OperationalDataService` folder, and open the `BizTalkOperationalData.pbit` file:
+2. Open the `\Program Files (x86)\Microsoft BizTalk Server\OperationalDataService` folder, and open the `BizTalkOperationalData.pbit` file:
 ![Open pbit file](../core/media/operational-data-pbit.png)
```

Comment:
Are you planning to update this image as well?

Reply:
I think we can just remove this picture, it doesn't really help. People should know how to open a file, they don't need to a guide on that. unless we are trying to emphasis something else.

Reply:
agree